news/2022-11-09-featured-artist-track-updates-monstercat.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: "Featured Artist Track Updates: Monstercat"
+date: 2022-11-09 11:00:00 +0000
+---
+
+A truckload of new **Monstercat** tracks have been delivered to our Featured Artist collection. Come check what's new!
+
+![](https://assets.ppy.sh/artists/255/header.jpg)
+
+We're going off-script today. Usually we showcase a new Featured Artist every Wednesday and Saturday, but we have a ton of new **Monstercat** tunes to throw at you, and almost all of them come from different artists.
+
+Instead of creating a bunch of Featured Artist listings with 1 song each, we decided to slam all **16** new songs onto the [**Monstercat**'s listing](https://osu.ppy.sh/beatmaps/artists/255)!
+
+For convenience, here's a full list of every newly added tune:
+
+- **7 Minutes Dead - The Follower**
```

Comment:
i think the bold's fine, it is the bulk of the content in the article for people who just wanna know what's available (especially since there's so many artists)